src/quacc/recipes/espresso/bands.py

```diff
@@ -43,14 +45,22 @@ def bands_pw_job(
     **calc_kwargs,
 ) -> RunSchema:
     """
-    Function to carry out a basic bands calculation with pw.x.
+    Function to carry out a basic bands structure calculation with pw.x.
+
+    First perform a normal SCF calculation as above; then use this job
```

Comment:
Let's refer to the specific function since if someone views the docstring in something like VS code via a hover-over, it might not be visible immediately. Also, there is nothing above :)

You can do something like
```
[quacc.recipes.espresso.core.static_job][]
```

to have it automatically hyperlink in the built docs.